sdk/tools/log2lines/options.c

```diff
@@ -166,8 +166,8 @@ int optionParse(int argc, const char **argv)
             break;
         case 'R':
             optCount++;
-            free(opt_Revision);
-            opt_Revision = malloc(LINESIZE);
+            if (!opt_Revision)
+                opt_Revision = malloc(LINESIZE);
```

Comment:
Note for reviewers: Now, the (already allocated) opt_Revision gets just re-used.

Reply:
I don't see the difference here - allocation size was always the same, and it is malloc and not calloc